### DISCUSSION TITLE
Mitigating duplicate issues for people new to GitHub

### DISCUSSION THREAD

**Discussion by lmsalustri** [TeamNewPipe/NewPipe#11315] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

This isn't related to the NewPipe app, but to the repo itself, so I'm asking here. I'm noticing that most of the people duplicating known issues such as #11191 are new GitHub accounts that were created just to report the issue through the Report on GitHub button. I see that replying to existing issues is limited to accounts that aren't new, but the same does not go for creating new issues. Would limiting the creation of new issues help mitigate duplicates? Or should known issues be added to the FAQ?

**Comment by opusforlife2** [comment]
> new GitHub accounts that were created just to report the issue through the Report on GitHub button

The mindset here is usually "AAAAA my stuff broke fix it ASAP". Open source etiquette is something to be learned over time. It's not something that comes naturally to people. The ideal behaviour of checking out the Readme, pinned issues, etc. has to be inculcated on purpose. FAQs or anything else won't make a difference.

The most in-your-face instruction is the template. If people ignore that, then we just close the issue. After all,

> I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.

:D

> I see that replying to existing issues is limited to accounts that aren't new

This isn't the case. Where have you seen this?

**Reply by lmsalustri** [reply]
> This isn't the case. Where have you seen this?

Experienced it myself. When my account was new, my ability to interact on this repo was limited for 24 hours.

**Reply by opusforlife2** [reply]
We temporarily limited interaction when everyone came in to complain about the YT issue.

**Reply by lmsalustri** [reply]
Well then I guess I just so happened to come in at the same time lol. Thanks for clearing that up!
